apis/cache/v1beta1/replication_group_types.go

```diff
@@ -244,7 +244,7 @@ type ReplicationGroupParameters struct {
 	//
 	// If false, changes to the nodes in the replication group are applied on the
 	// next maintenance reboot, or the next failure reboot, whichever occurs first.
-	ApplyImmediately bool `json:"applyImmediately"`
+	ApplyModificationsImmediately bool `json:"applyModificationsImmediately"`
```

Comment:
I guess we have some wiggle room on this naming because it's not really part of the actual ReplicationGroup API object, but rather one of the operations on it. This seems reasonable to me.

https://docs.aws.amazon.com/AmazonElastiCache/latest/APIReference/API_ModifyReplicationGroup.html

Reply:
> it's not really part of the actual ReplicationGroup API object, but rather one of the operations on it.

Yeah, actually this goes for some other fields as well. The actual `ReplicationGroup` object hides a lot of properties that you can actually control through operation objects. So, our API object is actually the sum of all of the possible configurations.